src/action.js

```diff
@@ -17,6 +17,8 @@ async function exportSecrets() {
     const secretsInput = core.getInput('secrets', { required: false });
     const secretRequests = parseSecretsInput(secretsInput);
 
+    const secretEncoding = core.getInput('secretEncoding', { required: false });
```

Comment:
Does anything validate that the given encoding is one of "base64", "hex", "utf8"? Are their other encodings that could work via the [buffer.From](https://nodejs.org/api/buffer.html#static-method-bufferfromstring-encoding) API that we'd want to prevent from using?

Reply:
Good note.  I will add a check for supported encoding type.